widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -274,6 +254,27 @@ public ChatController(MediaUpgradeOfferRepository mediaUpgradeOfferRepository,
         this.isShowOverlayPermissionRequestDialogUseCase = isShowOverlayPermissionRequestDialogUseCase;
     }
 
+    public void setPhotoCaptureFileUri(Uri photoCaptureFileUri) {
+        this.photoCaptureFileUri = photoCaptureFileUri;
+    }
+
+    public Uri getPhotoCaptureFileUri() {
+        return this.photoCaptureFileUri;
+    }
+
+    private final Observer fileAttachmentObserver = new Observer() {
+        @Override
+        public void update(Observable observable, Object o) {
+            if (viewCallback != null) {
+                viewCallback.emitUploadAttachments(getFileAttachmentsUseCase.execute());
+                emitViewState(chatState.setShowSendButton(isShowSendButtonUseCase.execute(chatState.lastTypedText)));
+            }
+        }
+    };
+
+    private final String TAG = "ChatController";
```

Comment:
variables can be placed on top of the class

Reply:
fixed